File: examples/python/processors/update.py
from dotenv import load_dotenv
from retab import Retab

assert load_dotenv("../../.env.production")

reclient = Retab()

processor = reclient.processors.update(
    processor_id="proc_F0FE8DFqyouQdZXDTWRg0",
    name="Invoice Processor Updated",
)

print(processor.model_dump_json(indent=2))

File: examples/node/processors/update.js
import { config } from 'dotenv';
import { AsyncRetab } from '@retab/node';

config();

const reclient = new AsyncRetab();

const processor = await reclient.processors.update({
  processor_id: "proc_F0FE8DFqyouQdZXDTWRg0",
  name: "Invoice Processor Updated",
});

console.log(JSON.stringify(processor, null, 2));